quickstarts/mlops/truera/dashboards/truera.json

```diff
@@ -0,0 +1,102 @@
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 6,
+            "row": 1,
+            "height": 3,
+            "width": 6
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "\n# Startup Guide\n\n## Initial bootstrap\n\n* Create and store Truera API key and New Relic API keys.\n* Install truera2newrelic tool.\n\n## Once per model:\n\n* For models to be monitored using new relic, stream metrics from truera to new relic using \"truera2newrelic metrics\" command. The data will show up in model score instability graph below.\n* Setup one dashboard in New Relic per ML model using \"truera2newrelic dashboard\" command. This dashboard will allow you to monitor data and analyze/diagnose issues.\n\nStep by step guide is part of [Trurea - New Relic integration doc](https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/mlops-integrations/truera-integration/)\n"
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 4,
+            "height": 1,
+            "width": 3
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "# [Monitor models in New Relic Dashboard](launcher/dashboards.launcher?platform[filters]=IihuYW1lIExJS0UgJ1RydWVyYSBVbmlmaWVkJyBPUiBpZCA9ICdUcnVlcmEgVW5pZmllZCcgT1IgZG9tYWluSWQgPSAnVHJ1ZXJhIFVuaWZpZWQnKSI=&platform[accountId]=3206889&pane=eyJuZXJkbGV0SWQiOiJkYXNoYm9hcmRzLmhvbWUifQ==)\n"
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 4,
+            "row": 4,
+            "height": 1,
+            "width": 3
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "## [Contact Truera team](https://truera.com/#elementor-action%3Aaction%3Dpopup%3Aopen%26settings%3DeyJpZCI6IjI2MDAiLCJ0b2dnbGUiOmZhbHNlfQ%3D%3D)"
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.line"
+          },
+          "layout": {
+            "column": 1,
+            "row": 5,
+            "height": 3,
+            "width": 10
+          },
+          "title": "Model Instability(NUMERICAL_WASSERSTEIN)",
+          "rawConfiguration": {
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 3206889,
```

Comment:
@vivekbhalgat all of these should be change to "accountId": 0,

Reply:
I have updated the json.